application/actions/requests.go

```diff
@@ -24,3 +28,49 @@ func convertRequestToAPIType(request models.Request) (api.Request, error) {
 
 	return output, nil
 }
+
+func convertRequestsToAPIType(requests []models.Request) ([]api.Request, error) {
+	output := make([]api.Request, len(requests))
+	
+	for i, request := range requests {
+		var err error
+		output[i], err = convertRequestToAPIType(request)
+		if err != nil {
+			return []api.Request{}, err
+		}
+	}
+
+	return output, nil
+}
+
+// getRequests responds to Get requests at /requests
+
+// go back to GraphQL code
```

Comment:
That was a comment that I've since removed (didn't mean to commit it) more a reminder to me to look at the request models code/graphql resolvers for guidance on fetching requests from the database.